embiggen/sequences/abstract_sequence.py

```diff
@@ -28,9 +29,12 @@ def __init__(
             Wthever to shuffle the vectors.
         elapsed_epochs: int = 0,
             Number of elapsed epochs to init state of generator.
+        seed: int = 42,
+            Random seed to make the sequence reproducible.
         """
         self._window_size = window_size
         self._shuffle = shuffle
+        self._seed = seed
 
         super().__init__(
             samples_number=samples_number,
```

Comment:
samples_number should be spelled ``sample_number``

Reply:
Fixed!